docs/html/development/getting-started.rst

```diff
@@ -49,18 +49,18 @@ pip's tests are written using the :pypi:`pytest` test framework, :pypi:`mock`
 and :pypi:`pretend`. :pypi:`tox` is used to automate the setup and execution of
 pip's tests.
 
+It is preferable to run the tests in **parallel** for better user experience during development,
+as the test suite is very extensive. WIthout the `-n auto` your tests would be running sequentially,
```

Comment:
Typo at _Without_.

Reply:
Acknowledged, would be fixed in new commit